### PR TITLE
add support to bitte wallet

### PR DESCRIPTION
**Pull request by sainthiago**
Checklist:

- [x]  I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

Summary
Bitte Wallet integration

PR summary
This PR contains the integration with Wallet selector of [bitte-wallet](https://wallet.bitte.ai/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,10 +73,12 @@ yarn add \
...
   @near-wallet-selector/ramper-wallet \
-  @near-wallet-selector/mintbase-wallet \
-  @near-wallet-selector/near-mobile-wallet \
+  @near-wallet-selector/near-mobile-wallet  \
   @near-wallet-selector/bitget-wallet \
-  @near-wallet-selector/okx-wallet
+  @near-wallet-selector/okx-wallet \
+  @near-wallet-selector/mintbase-wallet \
+  @near-wallet-selector/bitte-wallet
+
```
Comment (hcho112) [inline README.md:81]: extra empty line that need to be removed

**Review by hcho112 — CHANGES_REQUESTED** [review]
Apart from small chore change, rest looks ok. But I have a question, is Bitte Wallet a new mintbase wallet?

we already do have mint base wallet and if you were to create bitte wallet which reference same mintbase wallet codebase, what would be the point? 

If this is for purpose of rebranding, shouldn't this PR also prepare deprecation warning ?

**Comment by rubenmarcus** [comment]
> Apart from small chore change, rest looks ok. But I have a question, is Bitte Wallet a new mintbase wallet?
> 
> we already do have mint base wallet and if you were to create bitte wallet which reference same mintbase wallet codebase, what would be the point?
> 
> If this is for purpose of rebranding, shouldn't this PR also prepare deprecation warning ?

its kinda of re-branding but is not, the version of @mintbase-js/wallet package is different it was introduced in this P/R https://github.com/Mintbase/mintbase-js/pull/517/files, we want to deprecate Mintbase Wallet, but also still support due to many people being connected to the old servers, as also dapps that already use it.

File: README.md (hunk 1 above)
Comment (sainthiago) [inline README.md:81]:
```suggestion
```

Comment (sainthiago) [reply README.md:81]: updated :)

**Review by hcho112 — DISMISSED** [review]

**Review by hcho112 — APPROVED** [review]

**Comment by jackson-harris-iii** [comment]
@trechriron I'm having difficulty adding the bitte-wallet package as outlined in the [README.md](https://github.com/near/wallet-selector/tree/main/packages/bitte-wallet) and I noticed there isn't a release of the wallet-selector library to npm since prior to this update.

**Pull request merged**